README.md

```diff
@@ -5,12 +5,24 @@ parameters in a global, hierarchical key database._
 
 <img src="https://cdn.rawgit.com/ElektraInitiative/libelektra/master/doc/images/circle.svg" alt="Elektra" width="50" />
 
-The core is a small library implemented in C. The plugin-based framework fulfills many
-configuration-related tasks to avoid any unnecessary code duplication
-across applications while it still allows the core to stay without any
-external dependency. Elektra abstracts from cross-platform-related issues
-with a consistent API, and allows applications to be aware of other
-applications' configurations, leveraging easy application integration.
+Elektra provides a simple API for applications to access and update configuration data. The
+configuration data comes from configuration files that are _mounted_ into Elektra similar to 
+mounting a file system. As Elektra is a plugin based framework, there are a lot of _storage 
+plugins_ that support various configuration formats like ini, json, xml, etc. However, 
+there's a lot more to discover like executing python or lua scripts when a configuration
+value changes or enhanced validation plugins that won't allow corrupted configuration to 
+reach your application.
+
+As an application developer you get instant access to various configuration formats and the ability
+to fallback to a default configuration without having to deal with this on your own. As an administrator
+you can choose your favourite configuration format and _mount_ this configuration for the application.
+This features easy application integration as any application using Elektra can access any _mounted_ 
+configuration. You can even _mount_ your operating systems `hosts` or `fstab` configuration if needed,
+no need to configure the same data twice in different files.
```

Comment:
I also like this part, for `hosts` and `fstab` same concerns as above. Maybe: "`/etc` files such as `hosts` and `fstab`"?